net-misc/ubridge/files/ubridge-0.9.14-respect-flags.patch

```diff
@@ -0,0 +1,20 @@
+--- a/Makefile	2018-03-16 09:15:10.528245830 +0500
```

Comment:
I hope you're going to submit this upstream? ;-)

Reply:
> I hope you're going to submit this upstream? ;-)

Yes, I'll do it. Thanks.

Reply:
Upstream has already merged the patch.

https://github.com/GNS3/ubridge/pull/52